aptos-move/block-executor/src/executor.rs

```diff
@@ -570,9 +568,6 @@ where
                         txn_idx + 1,
                         scheduler.num_txns(),
                     );
-                    fail_point!("commit-all-halt-err", |_| Err(PanicOr::Or(
```

Comment:
I think we can remove this test and failpoint as is, what it currently tests (fallback when serialization error happens) is tested by another test unit test. I think it was supposed to pause one thread, so the block finished, and then make sure the error was still propagated. with the current flow, this is easier to see (no halt_ok and halt_err), but I can bring back the proper test later. Maybe add a TODO for me in unit_tests/mod.rs to write a proper test when you remove / disable https://github.com/aptos-labs/aptos-core/blob/ce593c4b0c9f066c821b80c403d6b5f8ec12263b/aptos-move/block-executor/src/unit_tests/mod.rs#L158

Reply:
Recalled why it was here, the test was actually working as desired, as this would only be invoked after the whole block was prepared for commit and scheduler was halted. It's okay to move it to the new place if we add a conditional to fail point return - or we can keep it here - added the comment to the new place.